.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: go
+go:
+  - 1.11.x
+
+os:
+  - linux
+
+script:
+  - go test -v -race -cover -timeout=1m ./...
```

Comment:
❓ Shouldn't we use the same configuration as in [Confita](https://github.com/heetch/confita/blob/master/.travis.yml)?

Reply:
At least > 1.10. WDYT?

Reply:
Sounds good 👍